src/inc/volatile.h

```diff
@@ -120,8 +120,22 @@ T VolatileLoad(T const * pt)
     STATIC_CONTRACT_SUPPORTS_DAC_HOST_ONLY;
 
 #ifndef DACCESS_COMPILE
+#if defined(_ARM64_) && defined(__GNUC__)
+    T val;
+    unsigned lockFreeAtomicSizeMask = (1 << 1) | (1 << 2) | (1 << 4) | (1 << 8);
+    if((1 << sizeof(T)) & lockFreeAtomicSizeMask)
+    {
+        __atomic_load((T volatile const *)pt, &val, __ATOMIC_ACQUIRE);
```

Comment:
On Arm64 These __atomics must be aligned.  I had tried to add an assert or an _ASSERTE, but each one caused different compilation errors.  

I hacked in an `if (unaligned) exit(17)` to debug.  Where I found the unaligned Volatile usage deleted below.

I think an assert would be appropriate.  If someone tells me how I will add one....